app/upgrades/v1.1.0/upgrades_test.go

```diff
@@ -165,6 +165,42 @@ func (suite *UpgradeTestSuite) TestAdminModuleUpgrade() {
 	suite.Require().Equal(uint64(1), id)
 }
 
+func (suite *UpgradeTestSuite) TestTokenFactoryUpgrade() {
+	var (
+		app                  = suite.GetNeutronZoneApp(suite.ChainA)
+		tokenFactorySubspace = app.GetSubspace(tokenfactorytypes.ModuleName)
+		ctx                  = suite.ChainA.GetContext()
+	)
+
+	var denomGasBefore uint64
+	tokenFactorySubspace.Get(ctx, tokenfactorytypes.KeyDenomCreationGasConsume, &denomGasBefore)
+	suite.Require().Equal(denomGasBefore, uint64(0))
+
+	// emulate mainnet state
+	tokenFactorySubspace.Set(ctx, tokenfactorytypes.KeyDenomCreationFee, sdk.NewCoins(sdk.NewCoin(params.DefaultDenom, sdk.NewInt(100_000_000))))
+
+	var denomFeeBefore sdk.Coins
+	tokenFactorySubspace.Get(ctx, tokenfactorytypes.KeyDenomCreationFee, &denomFeeBefore)
+	suite.Require().Equal(denomFeeBefore, sdk.NewCoins(sdk.NewCoin(params.DefaultDenom, sdk.NewInt(100_000_000))))
+
+	upgrade := upgradetypes.Plan{
+		Name:   v110.UpgradeName,
+		Info:   "some text here",
+		Height: 100,
+	}
+	app.UpgradeKeeper.ApplyUpgrade(ctx, upgrade)
+
+	var denomGas uint64
+	tokenFactorySubspace.Get(ctx, tokenfactorytypes.KeyDenomCreationGasConsume, &denomGas)
```

Comment:
during this migration, pfm params migrate from x/params to its own store, you need to get new params from the pfm keeper.